src/main/webapp/main/Index.js

```diff
@@ -0,0 +1,191 @@
+import React, { Component } from 'react';
+import { hh, h, div, h3, a } from 'react-hyperscript-helpers';
+import { About } from '../components/About';
+import { TableComponent } from "../components/TableComponent";
+import { IssueList, User } from "../util/ajax";
+import { parseDate } from "../util/TableUtil";
+import { Link } from 'react-router-dom';
+
+const columnsCopy = [{
+  dataField: 'project',
+  text: 'Project',
+  sort: true,
+  formatter: (cell, row, rowIndex, colIndex) => {
+    return div({}, [
+      linkFormatter(row, row.project)
+    ])
+  }
+}, {
+  dataField: 'title',
+  text: 'Title',
+  sort: true,
+  formatter: (cell, row, rowIndex, colIndex) => {
+    return div({}, [
+      linkFormatter(row, row.title)
+    ])
+  }
+}, {
+  dataField: 'status',
+  text: 'Status',
+  sort: true
+}, {
+  dataField: 'type',
+  text: 'Type',
+  sort: true
+}, {
+  dataField: 'updated',
+  text: 'Updated',
+  sort: true
+}, {
+  dataField: 'expiration',
+  text: 'Expiration',
+  sort: true
+}];
+
+const defaultSorted = [{
+  dataField: 'date',
+  order: 'desc'
+}];
+
+function linkFormatter(row, text) {
+  if (row.type === "Consent Group") {
+    return h(Link, {to: {pathname:'/newConsentGroup/main', search: '?consentKey=' + row.project, state: {issueType: 'consent-group', tab: 'review', consentKey: row.project}}}, [text])
+  }
+  return h(Link, {to: {pathname:'/project/main', search: '?projectKey=' + row.project, state: {issueType: 'project', tab: 'review', projectKey: row.project}}}, [text])
+}
+
+const Index = hh(class Index extends Component{
```

Comment:
I don't think naming this component as Index is a good approach. Maybe Main.js or LandingPage.js  ...